server/app/services/triplers.js

```diff
@@ -342,6 +342,7 @@ function calculateQueryPoints(query) {
  */
 function buildTriplerSearchQuery(req) {
   const {firstName, lastName, phone, distance, age, gender, msa} = req.query
+  const tripler_search_name_boost = ov_config.tripler_search_name_boost || 3 
```

Comment:
This || 3 seems like it might cause confusion. ov_config is setting the default to 2 if it's not set in the env vars, so the only effect I see this having is flipping the value to 3 if it was set to 0